<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting</class>
 <widget class="QWidget" name="Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="SaveBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>200</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CloseBtn">
   <property name="geometry">
    <rect>
     <x>421</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="FileEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>201</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="InerLab">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>60</y>
     <width>401</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="TitleLab">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>10</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="FileBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancleBtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>0</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>切换用户</string>
   </property>
  </widget>
  <widget class="QLabel" name="UserLab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>151</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>操作记录保存路径：</string>
   </property>
  </widget>
  <zorder>InerLab</zorder>
  <zorder>TitleLab</zorder>
  <zorder>SaveBtn</zorder>
  <zorder>CloseBtn</zorder>
  <zorder>FileEdit</zorder>
  <zorder>FileBtn</zorder>
  <zorder>label</zorder>
  <zorder>UserLab</zorder>
  <zorder>CancleBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
